pkg/scan/process_image.go

```diff
@@ -211,6 +212,10 @@ func isExecutable(path string) bool {
 	return fileMimetypeCheck(path, execMimeTypes)
 }
 
+func BytesToString(b []byte) (s string) {
```

Comment:
There is a reason golang calls this library unsafe !! Is there a reason we prefer this over string() or fmt.Sprintf ? @noboruma / @ibreakthecloud  ??

Reply:
`unsafe.String` is faster than using `string()`, since unsafe doesnot actually copies the data but instead creates a new string header that references the same underlying data as the byte slice. The down-side of this is any changes in byte slice will result in affecting the string, which is shouldn't be any trouble in our case.
cc: @noboruma

Reply:
one thing I am not sure if garbage collector will free this prematurely. @noboruma

Reply:
There are cases where GC can prematurely release the memory, but not in that scenario because the reference we have created is not unsafe anymore, it's a regular string and this will prevent GC from collecting further